bagitobjecttransfer/recordtransfer/views.py

```diff
@@ -58,6 +66,38 @@ class Index(TemplateView):
     template_name = "recordtransfer/home.html"
 
 
+def media_request(request, path: str) -> HttpResponse:
+    """Respond to whether a media request is allowed or not."""
+    if not request.user.is_authenticated:
+        return HttpResponseForbidden("You do not have permission to access this resource.")
+
+    if not path:
+        return HttpResponseNotFound("The requested resource could not be found")
+
+    user = request.user
+    allowed = False
+
+    if user.is_staff:
+        allowed = True
+
+    if not allowed:
+        return HttpResponseForbidden("You do not have permission to access this resource.")
```

Comment:
Perhaps this could be made more concise like this:
```suggestion
    if not user.is_staff:
        return HttpResponseForbidden("You do not have permission to access this resource.")
```